recipes/opene57/all/test_package/example.cpp

```diff
@@ -0,0 +1,376 @@
+/*
```

Comment:
I think example could be reduced, just to call one or couple of functions, but it's fine, up to you

Reply:
i will keep it so as it is since it's an original example from the forked library, I will submit a new example in the future if the API changes